guides/Releasing Expo Go for Android.md

```diff
@@ -31,12 +31,12 @@ This document will guide you through the process of releasing a new version of E
     **Why:** So that developers who used `expo-cli` to download Expo Go to their devices can download the update.
 
     **How:**
-    - Open CircleCI on the release branch and go to the `client` workflow. Once `client_android` job is finished, approve `client_android_apk_release_approve` job and follow the next job `client_android_apk_release` which takes and uploads the artifact archive from `client_android` job to staging.
+    - Open GitHub Checks for the release branch and go to the `Android Client / build` workflow. Once it's finished, run `et wd client-android-simulator` and follow that workflow run which takes and uploads the client APK to staging.
     - Test if this APK works as expected. Connect Android device to your computer or open up an emulator and then run expotools command `et client-install -p android` to install and launch an APK.
     - When you're ready to sync the versions change to production, run `et promote-versions`.
 
 5. **Submit the application to Play Store**
 
     **Why:** So that our users that downloaded Expo Go from Play Store can update easily.
 
-    **How:** Open the `client` workflow from which you downloaded `app-release.apk` in step 3. and approve the `client_android_approve_google_play` job. About 45 minutes later the update should be downloadable via Play Store.
+    **How:** Run `et wd client-android-release` which releases the APK to Play Store. About 45 minutes later the update should be downloadable via Play Store.
```

Comment:
I'd prefer using full names of expotools commands in docs, guides and CI workflows. Then it's more clear what the command that you copy-paste does. Aliases are mostly for people more familiar with expotools 😉 

```suggestion
    **How:** Run `et workflow-dispatch client-android-release` which releases the APK to Play Store. About 45 minutes later the update should be downloadable via Play Store.
```